include/seqan/misc/bit_twiddling.h

```diff
@@ -514,7 +514,7 @@ _bitScanForward(TWord word, WordSize_<NUM_BITS>)
     return _bitScanForwardGeneric(word, WordSize_<NUM_BITS>());
 }
 
-#if defined(PLATFORM_GCC)
+#if defined(COMPILER_GCC) || defined(COMPILER_INTEL) || defined(COMPILER_CLANG)
```

Comment:
* Test using `__builtin_clz`
```c++
#include <iostream>

int main() {
  unsigned msb = __builtin_clz(1 << (31 - 8));
  std::cout << "msb: " << msb << std::endl;
  return 0;
}
```

| os    | compiler | __builtin_clz |
--------| ---------- | ---------- | ---------------- |
| linux | gcc | ✔ 
| linux | clang | ✔
| linux | icc | ✔ 
| windows | msvc |  ✖ 
| windows | clang/llvm |  ✔
| windows | clang/c2 |  ✔ (cmake 3.6-rc) 
| windows | icc |  ✖ 

So windows icc and linux icc are not the same